### ISSUE TITLE
Not installing on RPi4

### ISSUE THREAD

**Issue by the-butterfry** [wjansenw/home-assistant-add-on#4]
This is from a supervised HA install logging. Not sure what is going on here 

21-04-23 06:27:45 INFO (SyncWorker_1) [supervisor.docker.addon] Starting build for 0665ed66/armv7-addon-grott:0.2
21-04-23 06:27:48 ERROR (SyncWorker_1) [supervisor.docker.addon] Can't build 0665ed66/armv7-addon-grott:0.2: The command '/bin/ash -o pipefail -c apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python' returned a non-zero code: 1
21-04-23 06:27:48 ERROR (SyncWorker_1) [supervisor.docker.addon] Build log: 
Step 1/22 : ARG BUILD_FROM
Step 2/22 : FROM $BUILD_FROM
 ---> 32848b90eab5
Step 3/22 : ENV LANG C.UTF-8
 ---> Using cache
 ---> b55c1960ccf7
Step 4/22 : ENV PYTHONUNBUFFERED=1
 ---> Using cache
 ---> 967856f3fd93
Step 5/22 : RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
 ---> Running in 3b6773948a2d
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/main/armv7/APKINDEX.tar.gz
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/main: temporary error (try again later)

fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/community/armv7/APKINDEX.tar.gz
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.13/community: temporary error (try again later)

ERROR: unable to select packages:

  python3 (no such package):
    required by: world[python3]
Removing intermediate container 3b6773948a2d

**Comment by johanmeijer** [comment]
@the-butterfry was this one already solved? 

Seems to happen more often with HA plugin installations (I am not running HA yet).

**Comment by the-butterfry** [comment]
no, i gave up. But it was random enough.. I have one HA (long existing old installation) that worked fine. The other new one that I ended up installing it on manually didn't work with HA for some reason. THat is, the one i wanted wouldn't install. Very fresh installation too. I didn't see much logging other than the attached above.

**Comment by charliesjc** [comment]
That's odd. I running it on HA and I had no problems. The docs on this repo were a bit light when it came to creating the addon. It took me a little while to realise I had to create an addon from scratch as far as HA was concerned and use the contents of this repo to do so. 

Maybe a silly question: you are running HA OS and not just the Core in Docker or something like that?
